index.html

```diff
@@ -0,0 +1,88 @@
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Email Template-2</title>
+  <style>
+    @import url('https://fonts.googleapis.com/css2?family=Anek+Malayalam&family=Lato:wght@400;700&family=Raleway:ital,wght@0,100;0,400;1,200&display=swap');
```

Comment:
In a template that we want to embed in email, its usually better if we don't have additional runtime imports. Do you really need these fonts? If not, I suggest we remove it. If we do need something specific, its better to copy/paste as local style to avoid run time fetching. 
It is possible that someone is reading an email that was delivered sometime back but when they are reading it locally, they don't have internet ON.

Reply:
The fonts have been removed